book/src/usage.md

```diff
@@ -66,6 +66,7 @@ Currently supported: `word`, `surround`, `function`, `class`, `parameter`.
 | `w`                    | Word                     |
 | `W`                    | WORD                     |
 | `(`, `[`, `'`, etc     | Specified surround pairs |
+| `m`                    | Closest surround pair    |
```

Comment:
Rather than doing this, I think doing something more specific is better, like `b` for `(`, `q` for `'` and similar, `m` is covers too much and the scope in too broad, user need to guess the context I believe to have this working the way they want.

Reply:
I've personally never needed this with vim-sandwich (from which I borrowed the `m` behavior), since it is supposed to be a catch-all -- that is it's sole purpose. If the closest pair isn't what I want I use the explicit form of `mi(`, etc instead of `mim`. The only drawback could be that we currently base the nearest pair matching off of the characters from the `PAIRS` constant which has some non-standard pairs:

https://github.com/helix-editor/helix/blob/1237b1a9b6861fa2049003d2e6a771f51ca9ca25/helix-core/src/surround.rs#L6-L14